content/departments/product-engineering/engineering/code-graph/code-insights/go_to_market.md

```diff
@@ -0,0 +1,66 @@
+# Code Insights go-to-market resources
+
+This is the source of truth list of go-to-market resources for the sales, customer engineering, and marketing teams. 
```

Comment:
Are you regularly getting these same questions from CE? I kind of think we're best handled separately, as our GTM content typically is specific to us.

Reply:
(I don't feel super strongly about this, and if it's easiest for you to maintain a single source of truth for the feature, we can note and fold this into the CE link structure.)

Reply:
I get a lot of "how can I demo" or "are there example templates for..." questions, and some implied questions (e.g. mistakes I see being made in demos on chorus calls that are covered in these resources, or not knowing about the demo templates covered in these resources). If you want to split this out or remove CE mentions, feel free! The resources here are still all applicable if it said "sales only" as well.

Reply:
Oh, that's good to know! In that case, wfm, happy to keep us here and link this within CE IA.

cc @IAmHughes

Reply:
Sounds good! I am not sure if the new learning system will cover CE stuff, but next on my list is perhaps a dedicated Code Insights version of https://handbook.sourcegraph.com/departments/product-engineering/engineering/code-graph/batch-changes/ce-onboarding/ – unless y'all don't use that page?

Reply:
It will, though where we fall in that list of prioritization I'm not 100%. But yeah, we'll be folding onboarding into that system with everyone else.

In terms of what people use, I'm honestly not sure—Process Street to start, but 100% there's a discoverability issue after initial onboarding. I added your page to our Education page.

Reply:
Thank you!